dask_ml/decomposition/pca.py

```diff
@@ -60,6 +60,21 @@ class PCA(sklearn.decomposition.PCA):
         improve the predictive accuracy of the downstream estimators by
         making their data respect some hard-wired assumptions.
 
+    center : bool, optional (default True)
+        When False (True by default), the underlying data gets centered at zero
+        by subtracting the mean of the data from the data itself.
+
+        PCA is performed on centered data due to its being a regression model,
+        without an intercept. As such, its pricipal components originate at the
+        origin of the transformed space.
+
+        `center` set to False may be employed when performing PCA on already
+        centered data.
+
+        Since centering is a required step as part of whitening, `center` set
+        to False and `whiten` set to True is a combination which may result in
+        unexpected behavior, if performed on not previously centered data.
+
```

Comment:
```suggestion
    center : bool, optional (default True)
        When True (the default), the underlying data gets centered at zero
        by subtracting the mean of the data from the data itself.

        PCA is performed on centered data due to its being a regression model,
        without an intercept. As such, its principal components originate at the
        origin of the transformed space.

        ``center=False`` may be employed when performing PCA on already
        centered data.

        Since centering is a required step as part of whitening, ``center`` set
        to False and ``whiten`` set to True is a combination which may result in
        unexpected behavior, if performed on not previously centered data.

```

I believe there are a few mistakes in this documentation as it's currently written.

* two backticks should be used for inline code formatting, since these docstrings are interpreted as [reStructuredText](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#inline-markup) by Sphinx
* the docstring should say that the default behavior (`center=True`) is to automatically center the data (it currently says the opposite)

Reply:
> the docstring should say that the default behavior (center=True) is to automatically center the data (it currently says the opposite)

Well spotted, thanks!
Updating as per the double backtick suggestion. I'll update the rest of the affected docstrings, to conform to the same guideline (unless we'd like to do that in a separate, dedicated PR?).

Reply:
If I was a maintainer here, I'd prefer that a docstring-only change be made in a separate pull request, and that this comment only apply to the lines you are already adding / changing as of this pull request